EvidenceApi.Tests/ConnectionString.cs

```diff
@@ -6,11 +6,7 @@ public static class ConnectionString
     {
         public static string TestDatabase()
         {
-            return $"Host={Environment.GetEnvironmentVariable("DB_HOST") ?? "127.0.0.1"};" +
-                   $"Port={Environment.GetEnvironmentVariable("DB_PORT") ?? "5432"};" +
-                   $"Username={Environment.GetEnvironmentVariable("DB_USERNAME") ?? "postgres"};" +
-                   $"Password={Environment.GetEnvironmentVariable("DB_PASSWORD") ?? "mypassword"};" +
-                   $"Database={Environment.GetEnvironmentVariable("DB_DATABASE") ?? "evidence_api_test"}";
+            return $"{Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? "Host=localhost;Port=3002;Database=evidence_api;Username=postgres;Password=mypassword"};";
```

Comment:
Changed because only needed to return a connection string or what the default set up for the docker compose is.